go.sum

```diff
@@ -0,0 +1,8 @@
+github.com/golang/protobuf v1.5.0/go.mod h1:FsONVRAS9T7sI+LIUmWTfcYkHO4aIWwzhcaSAoJOfIk=
```

Comment:
Why do we need go.sum in the root? We already have one inside src.

Reply:
Got it. That's to workaround go_package requiring '/'. Let's specify --go_opt=paths=source_relative instead when running protoc.